lua/telescope/mappings.lua

```diff
@@ -148,6 +148,8 @@ mappings.default_mappings = config.values.default_mappings
       ["<C-v>"] = actions.select_vertical,
       ["<C-t>"] = actions.select_tab,
 
+      ["<C-r>"] = actions.delete_file,
```

Comment:
doesnt make sense to map that on all pickers. Why would we want this for a picker like: `help_tags` ...

Reply:
agreed, should not be default mappings, but I'm OK with adding the action I think.